### ISSUE TITLE
"ARMED" notification blocks part of the display

### ISSUE THREAD

**Issue by cbf123** [yaapu/FrskyTelemetryScript#204]
If I force-arm the model via mission planner (I haven't calibrated the compass yet), it shows a red rectangle with the word "ARMED" near the middle of the display.  This blocks part of the artificial horizon.

Is this supposed to be there, or is it supposed to flash once or twice and then go away?

**Comment by cbf123** [comment]
Seems like maybe this is supposed to go away once actually flying?

**Comment by yaapu** [comment]
yes, as you probably have found out by yourself in the meantime :-)
